nixos/tests/agda.nix

```diff
@@ -0,0 +1,63 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  testfile = pkgs.writeText "TestModule.agda" ''
+    module TestModule where
+  '';
+  # Make sure this does not get put in the nix store as an agda-lib file
```

Comment:
Maybe link the corresponding agda issue here? Or this PR?

Reply:
agreed